src/utils/elasticDSL.js

```diff
@@ -79,10 +77,19 @@ let basicSimplifyTree = _.flow(
   ),
   // Rename __DOT__ to '.'
   tree => {
-    F.walk()(x => {
-      let dots = _.filter(_.includes('__DOT__'), _.keys(x))
-      _.each(dot => renameOn(dot, _.replace(/__DOT__/g, '.', dot), x), dots)
-    })(tree)
+    F.walk()(
+      x => {
+        let dots = _.filter(_.includes('__DOT__'), _.keys(x))
+        _.each(dot => renameOn(dot, _.replace(/__DOT__/g, '.', dot), x), dots)
+      },
+      // When pivot results are called, the results on interior node types aren't, so we need to ensure they are returned with the proper structure
```

Comment:
I don’t love how crazy this method is, but this is the right answer for now until we revisit and build a better alternative